Account.java

```diff
@@ -2,7 +2,7 @@
 
 public class Account {
 	
-	private String accountName;
+	public String accountName;
 	private double openingBalance;
```

Comment:
Consider using the mutators and accessors rather than making the properties public